bulletin_board/server/app/graphql/policies/visibility_for_all.rb

```diff
@@ -22,8 +22,12 @@ def self.trustee?(context)
       trustee = Client.where(unique_id: trustee_unique_id).first
       return false if trustee.blank?
 
-      decoded_api_key = JWT.decode(context[:api_key], trustee.public_key_rsa, true, verify_iat: true, algorithm: "RS256").first.with_indifferent_access
-      decoded_api_key[:trustee_unique_id] == trustee_unique_id
+      begin
+        decoded_api_key = JWT.decode(context[:api_key], trustee.public_key_rsa, true, verify_iat: true, verify_expiration: true, algorithm: "RS256").first.with_indifferent_access
```

Comment:
mm, there is no need to verify the `iat`, right?

Reply:
Dunno 😅 . Do you think we can remove this check?